recipe/meta.yaml

```
@@ -33,23 +33,25 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=61.0.0
```

```suggestion
    - setuptools
```
Pinning is redundant for python packages in host

---

The reason https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Recipe_standards.md#the-requirements-section